data/transport missions.txt

```diff
@@ -1412,6 +1415,139 @@ mission "Hallucination"
 
 
 
+mission "Shady passenger transport 1"
```

Comment:
Repeatable missions generally have the number in brackets (and are 0-indexed), while one-time missions do generally use unbracketed, 1-indexed numbers.

Reply:
I was following the Drug Running missions, which start at 1 and are unbracketed.

Reply:
That's fair. I didn't review @Amazinite 's pirate jobs pull or I would have commented this there too :grinning:

Reply:
That's fair. The game style has definitely evolved with time